changelog.d/14831.misc

```diff
@@ -0,0 +1 @@
+Non lazy loading sync not blocking during fast join.
```

Comment:
This could maybe be a `.feature`.

```suggestion
Faster joins: non lazy-loading syncs will return immediately after a faster join, by omitting partial state rooms until we acquire their full satte.
```